### PR TITLE
Ver 1.33 Add `invoices` and `invoiceTransaction` collections

### PR DESCRIPTION
**Pull request by dmccraw**

### CODE REVIEW

**Comment by dmccraw** [comment]
This is dependent on https://github.com/teamsnap/apiv3/pull/2932/files

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # TeamSnap JavaScript SDK CHANGELOG
 
+### May 12, 2017 // Version 1.23.0
```
Comment (amberwebb) [inline CHANGELOG.md]: Should this be after today's date and version 1.33.0 ?

File: docs/collections/invoice_transactions.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+## Methods
+
+- [loadInvoiceTransactions](#loadInvoiceTransactions)
+- [createInvoiceTransaction](#createInvoiceTransaction)
+- [saveInvoiceTransaction](#saveInvoiceTransaction)
+- [deleteInvoiceTransaction](#deleteInvoiceTransaction)
+
+
+---
+<a id="loadInvoiceTransactions"></a>
```
Comment (amberwebb) [inline docs/collections/invoice_transactions.md]: Should there be a couple of space above this?

File: CHANGELOG.md (hunk 1 above)
Comment (dmccraw) [inline CHANGELOG.md]: oops, copy and paste error

File: docs/collections/invoice_transactions.md (hunk 2 above)
Comment (dmccraw) [inline docs/collections/invoice_transactions.md]: Looks like the first <a> in each .md is right below the ---

**Comment by dmccraw** [comment]
@amberwebb updated

File: docs/collections/invoices.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+* `params`: [object] - an object with query parameters.
+* `callback`: [function] - callback to be executed when the operation completes.
+
+To see a list of all available search params you can run:
+`teamsnap.collections.invoices.queries.search.params`
+
+### Examples
+```javascript
+// ~~~~~
+// Loads all invoices for `id: 1`.
```
Comment (matthopson) [inline docs/collections/invoices.md]: Should that say "loads invoice for..." ?

File: docs/collections/invoice_transactions.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+* `params`: [object] - an object with query parameters.
+* `callback`: [function] - callback to be executed when the operation completes.
+
+To see a list of all available search params you can run:
+`teamsnap.collections.invoiceTransactions.queries.search.params`
+
+### Examples
+```javascript
+// ~~~~~
+// Loads all invoiceTransactions for `id: 1`.
```
Comment (matthopson) [inline docs/collections/invoice_transactions.md]: Should this say "load invoiceTransaction for..." ?

File: docs/collections/invoices.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+* `data`: [object, null] - data object to apply to the newly created object.
+
+### Examples
+```javascript
+// ~~~~~
+// Creates a new invoice item.
+var invoice = teamsnap.createInvoice();
+
+// ~~~~~
+// Creates a new invoice item with `batchInvoice: 1` and `memberId: 1`.
```
Comment (matthopson) [inline docs/collections/invoices.md]: `batchInvoiceId`?

File: docs/collections/invoices.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+var invoice = teamsnap.createInvoice({batchInvoice: 1, memberId: 1});
```
Comment (matthopson) [inline docs/collections/invoices.md]: Same as above

File: docs/collections/invoices.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+```
+
+
+---
+
+
+<a id="saveInvoice"></a>
+## `saveInvoice(invoice, callback)`
+Saves a `invoice` item.
```
Comment (matthopson) [inline docs/collections/invoices.md]: "an"

**Comment by dmccraw** [comment]
@matthopson Updated

**Pull request merged**